src/services/exprensesService.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const { Expense } = require('../models/expenses');
+
+function getAllExpenses() {
+  return Expense.findAll({
+    order: ['created_at'],
```

Comment:
You can write with camelCase createdAt as Sequelize automatically converts camelCase to snake_case.

Reply:
Done